src/App.js

```diff
@@ -14,23 +14,25 @@ import './App.css';
 
 const App = () => {
   return (
-    <MapCenter>
-      <Locations>
-        <Router>
-          <div className="app">
-            <main className="app__content">
-              <Switch>
-                <Route path="/" exact component={MapPage} />
-                <Route path="/list" component={ListPage} />
-                <Route path="/settings" component={Settings} />
-                <Route path="/help" component={Help} />
-              </Switch>
-            </main>
-            <Footer />
-          </div>
-        </Router>
-      </Locations>
-    </MapCenter>
+    <GoogleMaps>
```

Comment:
One nice thing is to give Context providers names like `GoogleMapsContextProvider` or `ThemeProvider` . That way at a glance we know exactly what that tag is doing.

Reply:
I second this. `GoogleMaps` is too generic of a name. `GoogleMapsProvider` would be the most idiomatic naming, from what I understand.

Reply:
_Thank you for your advice Lauren & Andrew! I'm so happy for them!_

I am going to create a new Jira ticket to fix it in the future 😄.

Reply:
[Jira Ticket](https://the-collab-lab.atlassian.net/browse/TCL30-49)  ^^ 😄